js/src/tooltip.js

```diff
@@ -371,22 +369,19 @@ class Tooltip extends BaseComponent {
     element.innerHTML = this._config.template
 
     const tip = element.children[0]
+    this.setContent(tip)
     tip.classList.remove(CLASS_NAME_FADE, CLASS_NAME_SHOW)
 
     this.tip = tip
     return this.tip
   }
 
-  setContent() {
-    const tip = this.getTipElement()
+  setContent(tip) {
     this._sanitizeAndSetContent(tip, this.getTitle(), SELECTOR_TOOLTIP_INNER)
   }
 
   _sanitizeAndSetContent(template, content, selector) {
     const templateElement = SelectorEngine.findOne(selector, template)
-    if (!templateElement) {
-      return
-    }
```

Comment:
You added this but then removed it again? 🤔

Reply:
Yes. And I left the commit to be seen.

It was rescuing the case, but it wasn't right to my eyes. 

The first commit was just checking if the template element inside popover was not existed, and for some oddie reason, on every `show()` execution was setting the content from the beginning.

The second commit checks and sets content once, only during tip creation.

Both are right. So, I am open to any opinions

Reply:
Hm..

I guess https://github.com/twbs/bootstrap/blob/47b21f6683de84900030b60b4d2808d52af856a4/js/src/tooltip.js#L396-L398 solves the same purpose sort of.

But If there is no `content` https://github.com/twbs/bootstrap/blob/47b21f6683de84900030b60b4d2808d52af856a4/js/src/tooltip.js#L386-L389 could throw an error.

Reply:
What if you do `if (!content && templateElement) {`?

Reply:
could be ok too. Preferred the less calculations approach

Reply:
I'm not saying to remove that calculations stuff.. What if someone messes with the template option?

Reply:
I guess is better throwing an error there, to let him know something is wrong, but you thoughts seems valid to. What do you propose?

Reply:
If you think 
![image](https://user-images.githubusercontent.com/2838324/127669370-1e1f7fff-8469-4225-96e1-08ec38b04e1c.png)
is helpful, then keep it like that.

Reply:
haah ok, you are persuasive